image/vision_transformer/hybrid_parallel/train_with_engine.py

```diff
@@ -13,13 +13,11 @@
 from colossalai.nn import CrossEntropyLoss
 from colossalai.nn.lr_scheduler import CosineAnnealingWarmupLR
 from colossalai.utils import is_using_pp
-from colossalai.engine.schedule import PipelineSchedule, NonPipelineSchedule
 from dataloader import DaliDataloader
-from model.vit import build_pipeline_vit
-from model_zoo.vit.vit import _create_vit_model
+from colossalai.utils.model.pipelinable import PipelinableContext
+from titans.model.vit.vit import _create_vit_model
```

Comment:
Is it possible to merge `hybrid_parallel` and `pipline_parallel` folders? `ViT_CIFAR10` should be merged as well since only the dataset is different.

Reply:
The pipeline_parallel has been merged into hybrid_parallel. Hybrid parallel with cifar10 dataset has been merged into hybrid_parallel folder as well, and data parallel with cifar10 has been merged into data_parallel folder.